src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        String[] fileInfo = workWithFile.readFile(fromFileName).split(System.lineSeparator());
+        String report = workWithFile.createReport(fileInfo);
+        workWithFile.writeFile(report, toFileName);
```

Comment:
1. Why do we create class instance within same class?
2. Please include command .split(System.lineSeparator()) to readFile() so it produces String[] instead of String. OR you may include it to createReport() so it accepts String instead of String[]. Methods of same class should match each other
```suggestion
        String[] fileInfo = readFile(fromFileName);
        String report = createReport(fileInfo);
        writeFile(report, toFileName);
```